arch/arm/soc/rda_5981/Kconfig

```diff
@@ -0,0 +1,23 @@
+#
+# Copyright (c) 2017 RDA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config SOC_FAMILY_RDA5981
```

Comment:
This is a really specific name for an SoC family. Is there a more general name you could use that could encompass additional SoCs in the family?